src/actions/actions_makeshift.py

```diff
@@ -23,12 +23,23 @@ class MakeshiftAction:
     MakeshiftAction("hover", "showHover", "editor.action.showHover"),
     MakeshiftAction("inspect", "showDebugHover", "editor.debug.action.showDebugHover"),
     MakeshiftAction(
-        "quick fix", "showQuickFix", "editor.action.quickFix", restore_selection=True
+        "quick fix",
+        "showQuickFix",
+        "editor.action.quickFix",
+        restore_selection=True,
+        post_command_sleep_ms=100,
     ),
     MakeshiftAction(
         "reference", "showReferences", "references-view.find", restore_selection=True
     ),
-    MakeshiftAction("rename", "rename", "editor.action.rename", restore_selection=True),
+    MakeshiftAction(
+        "rename",
+        "rename",
+        "editor.action.rename",
+        restore_selection=True,
+        await_command=False,
+        post_command_sleep_ms=150,
```

Comment:
I had to up the sleep time a bit. Otherwise it would fail on my computer sometimes. Now it seems to be really stable.

Reply:
Fine by me